vuepress/docs/v6.3.2/docs/releases/README.md

```diff
@@ -1,7 +1,93 @@
 
 ## New Features and Major Fixes
-This information will be provided when the next version of Entando is released.
 
-## Previous Releases
-Please see the `Versions` list in the main navigation menu above to access documentation and release notes for previous versions of Entando.
\ No newline at end of file
+### Bundler
+
+  - Now able to extract also the categories
+  - Optimized to download only the resources actually required by the extraction
+  - Now able to properly export the entire EntandoApp thanks to the "system level bundles" (see the ECR related note)
+
+### Infrastructure
+  - The deployment structure for an Entando application has been changed. In deployments prior to 6.3.2 an EntandoCompositeApp was deployed using a multi-container pod that contained the App Builder, App Engine, and Component Manager. Now each of these pods is deployed independently.
+
+### Entando Component Repository
+
+  - Implemented bundle plugin descriptor version 3, which by default implies routes without the plugin version in the url, so just organization + bundle name.
+    -  Note that the Component Generator doesn't support this descriptor version yet, but plugins can be easily adapted manually
+  - Introduced the concept of "System Level Bundles", which allows the installation of root level resources.
+  - Added  support the CMS categories
+  - Initial implementation of the bundle Installation plan.
+      - The Installation plan allows to inspect the content of a bundle and to resolve conflicts with the components already present in the destination EntandoApp
+  - Fixed support for bundles based on mysql
+  - Fixed bug that in some condition prevented the bundle plugin pod to properly restart when scaled down and up again
+  - Fixed intermittent issue on the operator that prevented installing new bundles until the operator was restarted
+
+### Entando Component Generator
+
+  - Fixed support for the plugin descriptor ingressPath
+  - Fixed native builds in the Quarkus blueprint
+
+### App Builder
+  - Fixed a couple of permission issues related to page editing.
+  - We started a process of optimization to improve the performance of AppBuilder on scenarios with a high number of pages and contents.
+  - The process will likely be completed for the next minor or path release (6.4.0 or 6.3.3)
+  - Added support for user Avatars
+  - Fixed the CONTENT SEARCH QUERY WIDGET which in some condition returned no results
+  - Several fixes in assets, content, page and user management
+
+### Entando CLI "ent"
+
+ - Ent prj to support profiles in backend build
+ - Ent prj to force semver on bundle versions (X.Y.Z or vX.Y.Z), in accordance with the rest of the installation process
+ - Ent prj now allows direct deployment of a bundle via `ent prj deploy`
+ - Implemented support for connection profiles and related connection to clusters via Kubernetes configuration files and/or context
```

Comment:
I think this is the same as the first bullet

Reply:
Looks different to me.  Not sure if they are the same.

Reply:
Just different language for the same thing. It's the ent profile command which lets you use kubeconfig/kubectx, etc. for specific remote servers. Not a big deal either way.